kafka/conn.py

```diff
@@ -135,8 +135,8 @@ def __init__(self, host, port, afi, **configs):
                 default: None.
             api_version (tuple): specify which kafka API version to use. Accepted
                 values are: (0, 8, 0), (0, 8, 1), (0, 8, 2), (0, 9), (0, 10)
-                If None, KafkaClient will attempt to infer the broker
-                version by probing various APIs. Default: None
+                If None, SimpleClient will attempt to infer the broker
```

Comment:
Both the new KafkaClient and the old SimpleClient use BrokerConnection, so we should probably describe just the BrokerConnection behavior here